packages/calcite-design-tokens/src/component/modal.json

```diff
@@ -3,260 +3,255 @@
       },
       "dark": {
-        "value": "$semantic.ui.color.text.3.dark",
+        "value": "{semantic.ui.color.text.3.dark}",
         "type": "color"
       }
     },
     "border": {
       "default": {
         "light": {
-          "value": "$semantic.ui.color.border.3.light",
+          "value": "{semantic.ui.color.border.3.light}",
           "type": "color"
         },
         "dark": {
-          "value": "$semantic.ui.color.border.1.dark",
+          "value": "{semantic.ui.color.border.1.dark}",
           "type": "color"
         }
       },
       "top": {
         "brand": {
           "light": {
-            "value": "$semantic.ui.color.brand.default.light",
+            "value": "{semantic.ui.color.brand.default.light}",
             "type": "color"
           },
           "dark": {
-            "value": "$semantic.ui.color.brand.default.dark",
+            "value": "{semantic.ui.color.brand.default.dark}",
             "type": "color"
           }
         },
         "info": {
           "light": {
-            "value": "$semantic.ui.color.info.default.light",
+            "value": "{semantic.ui.color.info.default.light}",
             "type": "color"
           },
           "dark": {
-            "value": "$semantic.ui.color.info.default.dark",
+            "value": "{semantic.ui.color.info.default.dark}",
             "type": "color"
           }
         },
         "success": {
           "light": {
-            "value": "$semantic.ui.color.success.default.light",
+            "value": "{semantic.ui.color.success.default.light}",
             "type": "color"
           },
           "dark": {
-            "value": "$semantic.ui.color.success.default.dark",
+            "value": "{semantic.ui.color.success.default.dark}",
             "type": "color"
           }
         },
         "danger": {
           "light": {
-            "value": "$semantic.ui.color.danger.default.light",
+            "value": "{semantic.ui.color.danger.default.light}",
             "type": "color"
           },
           "dark": {
-            "value": "$semantic.ui.color.danger.default.dark",
+            "value": "{semantic.ui.color.danger.default.dark}",
             "type": "color"
           }
         },
         "warning": {
           "light": {
-            "value": "$semantic.ui.color.warning.default.light",
+            "value": "{semantic.ui.color.warning.default.light}",
             "type": "color"
           },
           "dark": {
-            "value": "$semantic.ui.color.warning.default.dark",
+            "value": "{semantic.ui.color.warning.default.dark}",
             "type": "color"
           }
         }
       }
     },
     "background": {
-      "light": {
-        "value": "$semantic.ui.color.foreground.1.light",
-        "type": "color"
-      },
-      "dark": {
-        "value": "$semantic.ui.color.foreground.1.dark",
-        "type": "color"
-      }
+      "value": "{semantic.ui.color.foreground.1}",
```

Comment:
I noticed this was the only component-level token that has a description. Is it intentional? If so, will others be updated as well?

On a similar note, this was the only one I found where `background` entry that didn't have a `default` nor `light/dark` keys right under it.

Reply:
component tokens haven't been fully updated yet